docs/concepts/README.md

```diff
@@ -1,5 +1,6 @@
 ## Concepts
 
+* [Actions](actions.md)
```

Comment:
🙏

Reply:
I feel this is needed, but it's weird we are talking only about actions and not about state or the view. 

If actions get their own section on the docs, then the explanation in hello-world.md needs to be simplified, since it is currently trying to explain everything about actions.